docs/tutorial/basic-dl-query.md

```diff
@@ -0,0 +1,46 @@
+# DL query
+
+This tutorial is based off https://ontology101tutorial.readthedocs.io/en/latest/DL_QueryTab.html
+Created by: Melissa Haendel, Chris Mungall, David Osumi-Sutherland, Matt Yoder, Carlo Torniai, and Simon Jupp
+
+## DL query tab
+
+The DL query tab shown below provides an interface for querying and searching an ontology. The ontology must be classified by a reasoner before it can be queried in the DL query tab.
+
+For this tutorial, we will be using cc.owl which can be found [here](https://github.com/OHSUBD2K/BDK14-Ontologies-101/tree/master/BDK14_exercises/basic-dl-query).
```

Comment:
Please move any such file dependencies in obook itself, by creating a `dl-query` folder in the `tutorial` folder and place any ontologies or other resources in there. OBOOK should be as self-contained as possible.

Reply:
pulled over all the files in https://github.com/OBOAcademy/obook/pull/116, once that is merged I'll redirect this